docs/plugins.md

```diff
@@ -286,7 +287,8 @@ You can use this to check if you have enough permissions for the service to work
 
 The hook functions receive a `context` argument with some metadata and utility methods.
 
-- `.state`: A plain empty object that will be shared and passed to all hooks in the same recording process. It can be useful to persist data between the different hooks.
+- `.state`: An object that will be shared and passed to all hooks in the same recording process. It can be useful to persist data between the different hooks.
+    - `state.persistedState`: An object under `state` which should only contain serializable fields. It will be passed to the `cleanUp` hook if Kap doesn't shut down correctly. Use this to store fields necessary to clean up remaining effects.
```

Comment:
```suggestion
    - `state.persistedState`: An object under `state` which should only contain serializable fields. It will be passed to the `cleanUp` hook if Kap didn't shut down correctly last time it was run. Use this to store fields necessary to clean up remaining effects.
```